tox.ini

```diff
@@ -182,24 +185,100 @@ deps =
 commands =
     "{toxinidir}/.travis/environment"
 
-    "{toxinidir}/.travis/mypy" --config-file="{toxinidir}/tox.ini" {posargs:src}
+    mypy                                       \
+        --config-file="{toxinidir}/tox.ini"    \
+        --cache-dir="{toxworkdir}/mypy_cache"  \
+        {tty:--pretty:}                        \
+        {posargs:src}
 
 
 [mypy]
 
 # Global settings
 
-warn_redundant_casts = True
-warn_unused_ignores = True
-strict_optional = True
-show_column_numbers = True
+disallow_untyped_defs    = True
```

Comment:
Somewhere in here we probably want `disallow_any_generics` but this is 100% not a blocker.

Reply:
added to the TODO list